samples/ORKCatalog/ORKCatalog/Tasks/TaskListRow.swift

```diff
@@ -787,6 +797,7 @@ enum TaskListRow: Int, CustomStringConvertible {
         
         return ORKOrderedTask(identifier: String(Identifier.ConsentTask), steps: [
             visualConsentStep,
+            passcodeConsentStep,
```

Comment:
Can you make this `passcodeStep` to be the last step, normally password creation is after consent .